packages/react-native-sdk/src/components/Modal.tsx

```diff
@@ -0,0 +1,112 @@
+import type { ReactNode } from "react";
+import { Modal as RNModal, View, StyleSheet, TouchableOpacity, SafeAreaView, ScrollView } from "react-native";
```

Comment:
`react-native` included Modal might not work for majority of apps. It might get hidden behind native navigation elements from `react-native-screens`.